src/Extension/FluentExtension.php

```diff
@@ -282,7 +282,7 @@ public function augmentSQL(SQLSelect $query, DataQuery $dataQuery = null)
         }
 
         // On frontend only show if published in this specific locale
-        if (FluentState::singleton()->getIsFrontend()) {
+        if ($this->owner->config()->get('frontend-publish-required') && FluentState::singleton()->getIsFrontend()) {
```

Comment:
Config can't have `-` in it, switch to underscore.

Set it to default true with a `private static $frontend_publish_required = true;` in this same extension.

Reply:
Thanks, @tractorcow . I will update.